src/Features/Core/Portable/EmbeddedLanguages/RegularExpressions/LanguageServices/RegexOptionsStorage.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.Features.EmbeddedLanguages.RegularExpressions.LanguageServices;
+
+internal static class RegexOptionsStorage
+{
+    public static PerLanguageOption2<bool> ReportInvalidRegexPatterns = new(
+        "dotnet_unsupported_report_invalid_regex_patterns",
+        defaultValue: true,
+        isEditorConfigOption: true);
```

Comment:
and yjust to make sure.  a user can still turn this  on/off in the VS ui?

Reply:
Yes. Via Tools > Options.

Reply:
(but not via editorconfig UI)